runtime/sema/type.go

```diff
@@ -454,6 +455,8 @@ type OptionalType struct {
 	Type Type
 }
 
+var _ Type = &OptionalType{}
```

Comment:
Lines like this one are code cleanup: we intend these types to match these interfaces so it's desirable (IMO) to make that relationship explicit. My understanding is that the compiler optimizes away these statements so the only effect is a compile-time error if the type or the interface change.

Reply:
Correct! Thank you for adding these 👍